css/index.css

```diff
@@ -104,14 +127,53 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
```

Comment:
good job removing unnecessary properties that could interfere with your new stylings